test/int/breakOnLoad.test.ts

```diff
@@ -100,7 +100,11 @@ function runCommonTests(breakOnLoadStrategy: BreakOnLoadStrategy) {
 
             if (breakOnLoadStrategy === 'instrument') {
                 await launchWithUrlAndSetBreakpoints(url, testProjectRoot, scriptPath, bpLine, bpCol);
-                await dc.assertStoppedLocation('breakpoint', { path: scriptPath, line: bpLine, column: bpCol });
+                if(testSetup.isThisV1) { // V1 says debugger_statement
+                    await dc.assertStoppedLocation('debugger_statement', { path: scriptPath, line: bpLine, column: bpCol });

```

Comment:
Would it be better to do?
const expectedReason = testSetup.isThisV1 ? 'debugger_statement' : 'breakpoint';
await dc.assertStoppedLocation(expectedReason, { path: scriptPath, line: bpLine, column: bpCol });

Reply:
Done